### DISCUSSION TITLE
Problem with shinyuieditor package

### DISCUSSION THREAD

**Discussion by geaft** [community/community#53298] [Packages]
### Select Topic Area

Bug

### Body

Hallo,
i launched shinyuieditor and after some test to create dashboard it was blocked and show me the follow message:

Something went wrong rendering grid_page()
Error message:

Invalid layout definition. Not consistant number of columns in every row

Can you help me to solve this problem?
Thanks a lot for your assistance
![2023-04-19_16h16_49](https://user-images.githubusercontent.com/131176380/233362393-4d64700d-7f20-4809-876f-9b335dcb4b2d.png)

**Comment by rmdhfz** [comment]
Hi @geaft ,

The error message you received indicates that there is an issue with the layout definition of your Shiny UI dashboard. Specifically, the number of columns in each row of your layout definition is inconsistent.

To solve this problem, you will need to review the code for your dashboard layout and ensure that each row has the same number of columns. Make sure that you have not missed any closing parentheses or brackets that may be causing the issue.

If you are still having trouble, you may want to try simplifying your layout by starting with a basic template and building up from there. Additionally, you can check the documentation for Shiny UI to ensure that you are following best practices for dashboard layout design.

**Reply by geaft** [reply]
Thanks a lot for your assistance Hafiz,
i tried to create a dashboard for the first time with this builder and for simplicity' i removed the app code generated.
Now i think i take a look at the relative documentation to do much more better.
Have a nice day.
